crates/sui-core/src/authority/authority_store_pruner.rs

```diff
@@ -375,6 +389,8 @@ impl AuthorityStorePruner {
         let mut checkpoints_to_prune = vec![];
         let mut checkpoint_content_to_prune = vec![];
         let mut effects_to_prune = vec![];
+        let max_checkpoints_in_batch = config.max_checkpoints_in_batch.unwrap_or(10);
```

Comment:
nit: I wonder if it may help with readability if instead of making these config an Option type, we use the serde default function to explicitly set the default value in the config struct, so that future reader can quickly see the default value.

When I first read the code few weeks back, it took me quite sometime to realize that AuthorityStorePruningConfig::Default() is not really the default that will be applied in prod. But rather, the value in each `unwrap_or()` in the code where the config is accessed is.